.github/workflows/build.yml

```diff
@@ -18,6 +18,11 @@ jobs:
         sudo apt update -y
         sudo apt install -y libgtest-dev libgflags-dev openssl libssl-dev protobuf-compiler protoc-gen-go golang-go cmake
 
+
+    - name: test-code-formatting
+      run: |
+        ./CI/scripts/check-gofmt.sh
```

Comment:
This is the new call in CI builds to verify that `gofmt`-formatting is not regressing in future.